core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -3411,6 +3432,36 @@ payment.account.amazonGiftCard.addCountryInfo={0}\n\
   This will not affect your account age status.
 payment.amazonGiftCard.upgrade.headLine=Update Amazon Gift Card account
 
+payment.swift.info=Please make sure to include as much information as possible for SWIFT payments. Not all the information \
+may be required for some payments, however as SWIFT is international you will not know from which country your offer is \
+taken and therefore, what information is needed. As such please provide as much information to make the process as smooth \
+as possible. \n\
+\n\
+Most banks charge fees for sending and receiving SWIFT payments. Bisq requires BTC Buyers to send SWIFT payments using \
+the shared fee model (SHA). If you send a payment using another mode for example Beneficiary pays fees (BEN) you will \
+likely be penalized. Therefore, please take care to select the correct fee model when sending payment. \n\
+\n\
+SWIFT payments require more information than any other payment method on Bisq. Please ensure you communicate with your \
+trading peer effectively and promptly to ensure a smooth trading experience for all. As is the case for all payment \
+methods please do leave any reference blank. If a reference is required please use your 'Account Owner Name'.
+
+payment.swift.info.buyer=Most banks charge fees for sending and receiving SWIFT payments. Bisq requires BTC Buyers to \
+send SWIFT payments using the shared fee model (SHA). If you send a payment using another mode for example Beneficiary \
+pays fees (BEN) you will likely be penalized. Therefore, please take care to select the correct fee model when sending \
+payment and make yourself aware of what fees you will likely pay in advance of taking the trade.\n\
+\n\
+SWIFT payments require more information than any other payment method on Bisq. Please ensure you communicate with your \
+trading peer effectively and promptly to ensure a smooth trading experience for all. As is the case for all payment \
+methods please do leave any reference blank. If a reference is required please use your 'Account Owner Name'.
```

Comment:
```suggestion
payment.swift.info.buyer=To buy bitcoin with SWIFT, you must:\n\
\n\
- send payment in the currency specified by the offer maker \n\
- use the shared fee model (SHA) to send payment\n\
\n\
Please review further guidance on the wiki to avoid penalties and ensure smooth trades [HYPERLINK:https://bisq.wiki/SWIFT].
```